docs/use_cases/personalized_search.md

```diff
@@ -1,15 +1,98 @@
+
+In this digital age, organizations across the globe are grappling with an exponential surge in data. By 2025, the world is projected to have over 175 zettabytes of data, [source](https://www.forbes.com/sites/tomcoughlin/2018/11/27/175-zettabytes-by-2025). This data, both structured and unstructured, is a treasure trove of insights waiting to be unlocked. The key to this treasure chest? Personalized search using vector embeddings.
+
+## What is a vector embedding?
+
+Vector embeddings are revolutionizing the way we search and retrieve information. They work by converting data into a numerical representation, known as a vector. This process allows the search system to consider the semantics, or the underlying meaning, of the data when performing a search.
+
+![Illustration of vector embeddings](../assets/use_cases/personalized_search/embeddings.png)
+
+For instance, consider a user searching for a book on an online store. With traditional keyword-based search, the user would need to know the exact title or author's name. But with vector embeddings, the user could simply describe the book's theme or plot, and the search system would retrieve relevant results. This is because vector embeddings understand the 'meaning' of the query, rather than just matching keywords.
+
+### How does it search?
+
+The power of vector embeddings lies in their ability to quantify the similarity between two vectors. This is done using a distance metric, with cosine similarity being one of the most commonly used metrics. Simply put, cosine similarity measures how close two vectors are to each other, helping to determine how similar two pieces of data are. It returns relevant results even when the exact terms aren't present in the query. This is made possible by the general-purpose nature of vector embeddings, which can represent almost any form of data, from text to images to audio.
+
+When vectorizing data, it's crucial to remember that embedding models have a limit on the maximum input length. The twelve best-performing models, based on the [Massive Text Embedding Benchmark (MTEB)](https://huggingface.co/spaces/mteb/leaderboard), are limited to an input size of 512 tokens, while the 13th best has an exceptional input size limit of 8192 tokens.
+
+For example, consider a user searching for a news article on a particular topic. If the article is longer than the model's maximum context length, the search system would need to split the article into smaller pieces or use a sliding window approach to capture the entire document in a single vector. This ensures that the search results are accurate and relevant, regardless of the length of the source document.
+
+## But it's not only text that you can search!
+
+Contrary to popular belief, vector embeddings are not only applicable to unstructured data. In fact, they have always been integral to deep learning models and can be used to derive insights from structured data as well.
```

Comment:
Is it possible to be more specific than "popular belief"?
Does this phrase add anything? If not, omit.